app/routes/interestgroups/components/InterestGroupList.tsx

```diff
@@ -1,18 +1,32 @@
 import { Button } from '@webkom/lego-bricks';
+import { useEffect } from 'react';
 import { Helmet } from 'react-helmet-async';
-import { Link } from 'react-router-dom';
+import { Link } from 'react-router-dom-v5-compat';
+import { fetchAllWithType } from 'app/actions/GroupActions';
 import { Content } from 'app/components/Content';
 import NavigationTab from 'app/components/NavigationTab';
-import type { ActionGrant, Group } from 'app/models';
+import { GroupType, type Group } from 'app/models';
+import { selectGroupsWithType } from 'app/reducers/groups';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
 import InterestGroupComponent from './InterestGroup';
 import styles from './InterestGroup.css';
 
 export type Props = {
-  actionGrant: ActionGrant;
   interestGroups: Array<Group>;
 };
```

Comment:
Haven't researched where this is used outside of this file, if it is used at all - but as it's not used in this file, and not relevant to this file any more it would be nice to yeet it

Reply:
`actionGrant` is still used below

```ts
  const actionGrant = useAppSelector((state) => state.groups.actionGrant);
```